tests fail anyway, but this at least allows them to run

Index: tests/mallocmock.c
--- tests/mallocmock.c.orig
+++ tests/mallocmock.c
@@ -53,10 +53,10 @@ void *realloc(void *ptr, size_t size)
 	_mallocmock_realloc_size += size;
 	return __libc_realloc(ptr, size);
 }
+#endif
 
 void mallocmock_reset()
 {
 	_mallocmock_malloc_size = 0;
 	_mallocmock_realloc_size = 0;
 }
-#endif
